homeassistant/components/switch/__init__.py

```diff
@@ -31,6 +32,7 @@
     discovery.services.BELKIN_WEMO: 'wemo',
     wink.DISCOVER_SWITCHES: 'wink',
     isy994.DISCOVER_SWITCHES: 'isy994',
+    camera.DISCOVER_SWITCHES: 'child_switch',
```

Comment:
Yeah having it in the switch file is ok, I cannot see a reason right now where one would use it without the switch component.